README.md

```diff
@@ -1,6 +1,9 @@
 # PyoMyo
 Python module for the Thalmic Labs Myo armband. 
 
+##here is a new example.
+
+
```

Comment:
Christine this code is awesome and elegant.

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.svg)](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome&nbsp; | &nbsp;**Tags:** Unreadable, Not secure, Efficient, Elegant